.github/workflows/tests.yml

```diff
@@ -12,8 +12,8 @@ jobs:
     runs-on: macos-latest
     steps:
       - uses: actions/checkout@v1
-      - name: Select Xcode 12.2
-        run: sudo xcode-select -s /Applications/Xcode_12.2.app
+      - name: Print Xcode version
```

Comment:
Is this fixing something specific you're facing?

Reply:
Rarely but sometimes older versions get deprecated. E.g 

https://github.com/actions/virtual-environments/issues/4060

Min version is 12.4 after this one.

Reply:
i believe this plugin has little dep on xcode and its implementations so it's fine to just rely on latest version for now until we face problems

Reply:
Discussed offline, see an example: https://github.com/bazel-ios/cocoapods-bazel/runs/4740577782?check_suite_focus=true

That said I'm ok with changing this for now and see how CI behaves

Reply:
Thx for sharing the example too @chenxiao0228!